README.md

```diff
@@ -1,2 +1,252 @@
-# jazz-concert
-This repo is for a jazz concert website which include the detailed description and information of the concert.
+# Jazz Concert: Capstone Project
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Personal-Portfolio] <a name="about-project"></a>
+
```

Comment:
- Kindly update the name of this project to the current one as this. The name given is different from the project name.